### ISSUE TITLE
UML: Package vs Virtual Folder

### ISSUE THREAD

**Issue by marcelvanthoff** [opengds/OpenGDS#26]
I like the concept of grouping classes into a package. When generating the code I somehow expected that these classes would also be grouped into the same Virtual Folder inside the project. 

Perhaps a nice idea?

**Comment by MikaelHolmstrom** [comment]
Interesting. So when you place classes in packages in a class diagram, you like the classes to be created under virtual folders in the project?!?
Do you prefer a real folder to be created on disc as well, or just a flat folder structure on disc?

**Comment by marcelvanthoff** [comment]
Yes, indeed. I created some packages and put in each a collection of classes. Either the same child or a nice structure of parents and children. As these are already grouped in a package in the UML, I thought it would be nice to group them as well in the project with virtual folders. 

Storing on disk in different folders would be an option. I like that idea too, but personally, that isn't so important to me.
